spk/sonarr/Makefile

```diff
@@ -0,0 +1,60 @@
+SPK_NAME = nzbdrone
```

Comment:
makes sense,
thanks, wasn't sure if we needed to maintain backwards compatibility.